paypal/express/views.py

```diff
@@ -289,6 +300,8 @@ def build_submission(self, **kwargs):
         submission['payment_kwargs']['payer_id'] = self.payer_id
         submission['payment_kwargs']['token'] = self.token
         submission['payment_kwargs']['txn'] = self.txn
+        print "PAYPALL SUBMISSIONN AFTERR", submission
```

Comment:
@monokrome  : we can ignore this PR.. There is another PR with the cleaned up code...  https://github.com/britco/django-oscar-paypal/pull/2